components/app/footer/FooterBlock.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="lg:flex lg:gap-10">
+    <a class="table mx-auto" :href="items[0].route" >
+      <div class="lg:flex lg:items-center lg:justify-center lg:gap-2">
+        <img
+          :src="items[0].icon"
+          :alt="items[0].name + ' logo'"
+          class="lg:w-6 lg:h-6 w-10 h-10 block mx-auto"
+        />
+        <p :class="`mt-0 lg:mt-0 font-semibold ${theme} lg:text-sm`">
+          {{ items[0].name }}
+        </p>
+      </div>
+    </a>
+
+    <div class="table mx-auto md:mt-0 mt-4">
+      <div class="grid sm:grid-cols-2 gap-2 sm:gap-10">
+        <a
+          :href="items[1].route"
+          rel="noopener noreferrer"
+          target="_blank"
+          class="table mx-auto"
+        >
+          <div class="flex items-center justify-center gap-2">
+            <img
+              :src="items[1].icon"
+              :alt="items[1].name + ' logo'"
+              class="w-6 h-6"
+            />
+            <p class="text-sm">{{ items[1].name }}</p>
+          </div>
+        </a>
+        <a
+          :href="items[2].route"
+          rel="noopener noreferrer"
+          target="_blank"
+          class="table mx-auto"
+        >
+          <div class="flex items-center justify-center gap-2">
+            <img
+              :src="items[2].icon"
+              :alt="items[2].name + ' logo'"
+              class="w-6 h-6"
+            />
+            <p class="text-sm">{{ items[2].name }}</p>
+          </div>
+        </a>
```

Comment:
suggestion(blocking): the footer layout is too static. We need to be able to add or remove items easily from the footer via array. Maybe use a `v-for` in combination with the `AppLink` component to resolve internal and external links.